scripts/prepdocs.py

```diff
@@ -314,6 +314,29 @@ def refresh_openai_token():
         openai.api_key = token_cred.get_token("https://cognitiveservices.azure.com/.default").token
         open_ai_token_cache[CACHE_KEY_CREATED_TIME] = time.time()
 
+"""Readfile function
```

Comment:
Please move inside the function and remove top line, ie.

def read_files(path_pattern, use_vectors):
  """
  Recursively read directory structure under `path_pattern`
  and execute indexing for the individual files
  """

Reply:
done